include/istio/control/http/controller.h

```diff
@@ -81,6 +84,9 @@ class Controller {
     // The LRU cache size for service config.
     // If not set or is 0 default value, the cache size is 1000.
     int service_config_cache_size{};
+
+    // local attributes
+    const ::istio::utils::LocalAttributes* local_attributes;
```

Comment:
It should be rare to hold a raw pointer since ownership is not clear.   Please use unique_ptr